x-pack/plugins/apm/public/components/app/service_map/Popover/stats_list.tsx

```diff
@@ -81,59 +140,31 @@ export function StatsList({ data, isLoading }: StatsListProps) {
     return <NoDataMessage />;
   }
 
-  const items = [
-    {
-      title: i18n.translate(
-        'xpack.apm.serviceMap.avgTransDurationPopoverStat',
-        {
-          defaultMessage: 'Latency (avg.)',
-        }
-      ),
-      description: isNumber(avgTransactionDuration)
-        ? asDuration(avgTransactionDuration)
-        : null,
-    },
-    {
-      title: i18n.translate(
-        'xpack.apm.serviceMap.avgReqPerMinutePopoverMetric',
-        {
-          defaultMessage: 'Throughput (avg.)',
-        }
-      ),
-      description: asTransactionRate(avgRequestsPerMinute),
-    },
-    {
-      title: i18n.translate('xpack.apm.serviceMap.errorRatePopoverStat', {
-        defaultMessage: 'Failed transaction rate (avg.)',
-      }),
-      description: asPercent(avgErrorRate, 1, ''),
-    },
-    {
-      title: i18n.translate('xpack.apm.serviceMap.avgCpuUsagePopoverStat', {
-        defaultMessage: 'CPU usage (avg.)',
-      }),
-      description: asPercent(avgCpuUsage, 1, ''),
-    },
-    {
-      title: i18n.translate('xpack.apm.serviceMap.avgMemoryUsagePopoverStat', {
-        defaultMessage: 'Memory usage (avg.)',
-      }),
-      description: asPercent(avgMemoryUsage, 1, ''),
-    },
-  ];
-
   return (
-    <table>
-      <tbody>
-        {items.map(({ title, description }) => {
-          return description ? (
-            <ItemRow key={title}>
+    <EuiFlexGroup direction="column" responsive={false} gutterSize="m">
+      {items.map(({ title, valueLabel, timeseries, color }) => {
+        if (!valueLabel) {
+          return null;
+        }
+        return (
+          <EuiFlexItem key={title}>
+            <EuiFlexGroup gutterSize="none" responsive={false}>
               <ItemTitle>{title}</ItemTitle>
```

Comment:
Instead of custom styles it would be better if we can handle this with EUI components

```suggestion
              <EuiFlexItem><EuiText color="subdued">{title}</EuiText></EuiFlexItem>
```